src/components/Asset/Edit/index.tsx

```diff
@@ -37,23 +37,23 @@ export default function Edit({ uri }: { uri: string }): ReactElement {
     }
   ].filter((tab) => tab !== undefined)
 
-  return asset && asset?.accessDetails && accountId === owner ? (
+  return !asset || !accountId || !owner || !asset?.accessDetails ? (
```

Comment:
we should not add another set of crazy long clauses here, and continue to use the loader as the fallback only so we don't have to add every possible condition here. All those conditions should live on the next clause to define if edit should be shown or not.

Reply:
I've removed this